community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/resume.py

```diff
@@ -134,8 +130,14 @@ def create_instances_request(nodes, partition_name, placement_group, job_id=None
     log.debug(f"create_instances_request: {model} placement: {placement_group}")
 
     body = NSDict()
+
     body.count = len(nodes)
-    body.minCount = 1
+
+    if placement_group:
```

Comment:
Isn't it simpler to just use
```
# Only set minCount if we need placement.
if not placement_group:
     body.minCount
```

Reply:
I want to explicitly spell out "no op" branch.